module4/projects/solo_stretch_tech/index.md

```diff
@@ -3,14 +3,137 @@ title: Solo Stretch Tech
 layout: page
 ---
 
```

Comment:
I put everything on one page, but feel free to split it up if you think that's better.